CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#10841](https://github.com/rubocop/rubocop/pull/10841): Optimize cache. ([@ChrisBr][])
```

Comment:
There should be a level 3 heading above this line. I guess it should be `### Changes`.

Reply:
The following PR template seems to be a good way to write a CHANGELOG.

> Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named {change_type}_{change_description}.md if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.